Source/WebCore/Modules/cookie-store/CookieListItem.h

```diff
@@ -36,15 +36,35 @@ namespace WebCore {
 struct CookieListItem {
     CookieListItem() = default;
 
-    CookieListItem(Cookie&& cookie)
+    CookieListItem(const Cookie& cookie)
+        : name(cookie.name)
+        , value(cookie.value)
+        , domain(cookie.domain)
+        , path(cookie.path)
+        , expires(cookie.expires)
+        , secure(cookie.secure)
     {
-        name = WTFMove(cookie.name);
-        value = WTFMove(cookie.value);
-        domain = WTFMove(cookie.domain);
-        path = WTFMove(cookie.path);
-        expires = cookie.expires.has_value() ? std::optional { *cookie.expires } : std::nullopt;
-        secure = cookie.secure;
+        switch (cookie.sameSite) {
```

Comment:
Should be moved to an inline function in the header to avoid code duplication.

Reply:
Since this new constructor is not needed, I have removed it and not inlined this block to a new function. Is this ok?